solutions/usaco-1059.mdx

```diff
@@ -6,6 +6,70 @@ author: Sathvik Chundru
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_prob1_bronze_dec20.html)
+## Video Solution
+
+By Jeffrey Meng
```

Comment:
wait also add to the authors in the yaml part

Reply:
OOPS